go/contacts/cache.go

```diff
@@ -157,33 +153,42 @@ func (c *CachedContactsProvider) LookupAll(mctx libkb.MetaContext, emails []keyb
 		conCache = newLookupResultCache()
 	} else {
 		mctx.Debug("Fetched cache, current cache size: %d", len(conCache.Lookups))
+		conCache.Version.Major = cacheCurrentMajorVersion
```

Comment:
?

also, where is the part where it throws away result if it's old major version?

Reply:
(in particular, don't we need to do that in this pr because we're changing the cache type?)

Reply:
I thought it didn't have to for this one, it will load expireAt to 0 for all entries and they will be refreshed (or deleted) next time LookupAll is performed. So it might be a minor inconvenience for people but for now we are only testing internally. Let me know if you think that's not the case.

Reply:
good point, it'd've been fine then